packages/browserify/test/generateConfig.js

```diff
@@ -11,26 +11,28 @@ test('generateConfig - empty policy', async (t) => {
   const scenario = createScenarioFromScaffold({
     defineEntry: () => {},
     defaultConfig: false,
+    expectedResult: { resources: {} }
   })
   const policy = await autoConfigForScenario({ scenario })
-  t.deepEqual(policy, { resources: {} }, 'policy matches expected')
+  scenario.checkResult(t, policy, scenario)
 })
 
 test('generateConfig - basic policy', async (t) => {
   const scenario = createScenarioFromScaffold({
     defineOne: () => { module.exports = global.two },
-    defaultConfig: false
-  })
-  const policy = await autoConfigForScenario({ scenario })
-  t.deepEqual(policy, {
-    resources: {
-      one: {
-        globals: {
-          two: true
+    defaultConfig: false,
+    expectedResult: {
+      resources: {
+        one: {
+          globals: {
+            two: true
+          }
         }
       }
     }
-  }, 'policy matched expected')
+  })
+  const policy = await autoConfigForScenario({ scenario })
+  scenario.checkResult(t, policy, scenario)
```

Comment:
dont use checkResult to assert an unrelated value 
just keep the original assertion